### PR TITLE
Add azure_network_watcher and azure_security_group_flow_logs

### PR DESCRIPTION
**Pull request by ndowmon**
This graph is a little messy with all of the diagnostic settings, but note the `azure_security_group_flow_logs` entity and its three relationships.

flowlog

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -253,6 +257,8 @@ The following relationships are created/mapped:
 | `azure_role_assignment`           | **ASSIGNED**          | `azure_user`                          |
 | `azure_role_assignment`           | **ASSIGNED**          | `azure_user_group`                    |
 | `azure_role_assignment`           | **USES**              | `azure_role_definition`               |
+| `azure_security_group_flow_logs`  | **USES**              | `azure_storage_account`               |
```
Comment (mknoedel) [inline docs/jupiterone.md:260]: Doesn't matter right now, but should this be reversed?
 `azure_storage_account` **USES** `azure_security_group_flow_logs`     ?

File: src/steps/resource-manager/network/index.test.ts [hunk 2]
```diff
@@ -1891,3 +1901,220 @@ describe('step = fetch azure firewalls', () => {
...
+        matchRequestsBy: getMatchRequestsBy({ config: configFromEnv }),
+      },
+    });
+
+    const {
+      accountEntity,
+      networkWatcherEntities,
+      securityGroupEntities,
+      storageAccountEntities,
+    } = await getSetupEntities(configFromEnv);
```
Comment (mknoedel) [inline src/steps/resource-manager/network/index.test.ts:2071]: Neat

**Review by mknoedel — APPROVED** [review]
image

File: docs/jupiterone.md (hunk 1 above)
Comment (ndowmon) [reply docs/jupiterone.md:260]: Good question. The flow logs just "use" this storage account for their underlying storage, making the flow logs `source` and storage account `target` in this relationship. As you can see, the the flow logs is the `target` in the other two relationships though.

**Pull request merged**